docs/guides.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: What is Battlesnake?
 sidebar_position: 0
+slug: /
```

Comment:
This makes this page the 'real' root page for the site, without needing a redirect!

Realized a con of this though is that the URL is changing meaning the old one with 404 😢 
If thats not desired I can roll this back!